vlselect/vlinsert: allow disabling the vlinsert and vlselect endpoints (#9067)
VictoriaMetrics/VictoriaMetrics: docs/victorialogs/cluster.md

Another option is to use third-party HTTP proxies such as [vmauth](https://docs.victoriametrics.com/victoriametrics/vmauth/), `nginx`, etc. for authorizing and encrypting communications
between VictoriaLogs cluster components over untrusted networks.

> By default, all the logs component (vlinsert, vlselect, vlstorage) are supports full HTTP endpoints including `/insert/*` and `/select/*`. It's recommended to disable select endpoints on `vlinsert` and insert endpoints on `vlselect`:

Reviewer: ```suggestion
By default, all the logs component (vlinsert, vlselect, vlstorage) support all the HTTP endpoints including `/insert/*` and `/select/*`. It's recommended to disable select endpoints on `vlinsert` and insert endpoints on `vlselect`:
```

```sh
# Disable select endpoints on vlinsert

> ./victoria-logs-prod -select.disable

Reviewer: ```suggestion
./victoria-logs-prod -storageNode=... -select.disable
```

# Disable insert endpoints on vlselect

> ./victoria-logs-prod -insert.disable

Reviewer: ```suggestion
./victoria-logs-prod -storageNode=... -insert.disable
```
